src/pml_vqvae/dataset/dataloader.py

```diff
@@ -2,33 +2,12 @@
 import torchvision
 import torch
 
-import numpy as np
 from pml_vqvae.visuals import show_image_grid
 
-from datasets import ImageNetDataset, CifarDataset
+from cifar10 import CifarDataset
+from imagenet import ImageNetDataset
 
-DATASET_DIR = "/home/space/datasets/"
-DATASET_NAMES = ["imagenet", "imagenet_mini", "cifar"]
-
-
-def get_random_indices(high: int, n: int, seed=None):
-    """Generates n random indices from 0 to high-1 without replacement
-
-    Args:
-        high (int): highest index
-        n (int): number of indices to generate
-        seed (int, optional): Seed for reproducibilty . Defaults to None.
-
-    Returns:
-        list: List of random indices
-    """
-
-    if seed:
-        rng = np.random.default_rng(seed=seed)
-    else:
-        rng = np.random.default_rng()
-
-    return rng.choice(high, n, replace=False)
+DATASET_NAMES = ["imagenet", "cifar"]
```

Comment:
This isnt part of this commit but so I'll just leave it here: Correct me if Im wrong, but it seems like the shuffle param in the load_data function is both applied for train and test split. I think there should be the possibility to obtain a shuffled train set and an unshuffled test split without calling that function twice

Reply:
Yes, we can do that. But does it matter if the test set is shuffled or not. I think it makes not difference on the result.

Reply:
No it doesn't change the result, but this way we can monitor how the model changes its reconstructions/prediction of a specific sample during training. If we for example validate our model every 10 epochs, we could the save a visualization of the reconstruction of the first batch of the test_loader, which if unshuffled would be always the same.

But yeah probably not that necessary

Reply:
well, therefore is the seed